### DISCUSSION TITLE
怎么改头像啊，有没有人教教我

### DISCUSSION THREAD

**Discussion by Faethwith** [share121/inter-knot#5077] [委托]
![Uploading 52030650832889801001725094549066_edit_816529832580092.jpg…]()

**Comment by cuijiaqi0406** [comment]
草我也是不会改

**Comment by KELERSAMA** [comment]
#4990 有教程

**Reply by Faethwith** [reply]
会了会了，谢谢

**Comment by Yizakl** [comment]
新人不要天天灌水qwq，多看看教程

**Reply by KELERSAMA** [reply]
ewq 大佬

**Reply by Yizakl** [reply]
我是fw喵

**Reply by KELERSAMA** [reply]
你帮助了iOS用户获得了神网APP 你是大佬 😭😡

**Reply by 00qio** [reply]
github，右上角头像，your profile，大头像下面有个edit profile就可以编辑了
